_includes/suggested-itinerary.html

```diff
@@ -0,0 +1,38 @@
+<!-- ITINERARY -->
+<section id="itinerary">
+  <div class="container">
+    <div class="row">
+      <div class="col-lg-12">
+        <h2 class="uppercase">Suggested Itinerary</h2>
+        <p class="lead">
+          While in Portugal, we'd recommend you to take the opportunity to get a taste of what it offers, like enjoying Porto's Midsummer festival!
+        </p>
+        <h4>Tuesday June 23</h4>
+        <p class="lead">
+          Arrive in Porto and explore the city! At night, Porto's Midsummer festival <a href="https://en.wikipedia.org/wiki/Festa_de_São_João_do_Porto">São João</a> will take place. It's a real experience, where you hit people on their heads with plastic hammers. 🔨<br />
+          Maybe grab a <a href="https://en.wikipedia.org/wiki/Francesinha">Francesinha</a> or XYZ for dinner?
+        </p>
+        <h4>Wednesday June 24</h4>
+        <p class="lead">
+          After recovering from São João, you can head down to Aveiro by train. Seafood here is great, so those dinner plans should be sorted!
+        </p>
+        <h4>Thursday June 25</h4>
+        <p class="lead">
+          Conference time! There'll be lunch, as well as an organized dinner and party in the evening. Time to learn and make friends!
+        </p>
+        <h4>Friday June 26</h4>
+        <p class="lead">
+          Day two of workshops! Lunch is again provided. Maybe find a nice restaurant in the city with some of those friends you made yesterday and today?
```

Comment:
For dinner, what about finding a nice restaurant in the city with some of those friends you made yesterday and today?